compose/ui/ui-text/src/jsNativeMain/kotlin/androidx/compose/ui/text/input/GapBuffer.jsNativeMain.kt

```diff
@@ -23,6 +23,6 @@ internal actual fun String.toCharArray(
     endIndex: Int
 ) {
     (startIndex until endIndex).forEach {
-        destination[destinationOffset + it] = get(it)
+        destination[destinationOffset + it - startIndex] = get(it)
```

Comment:
Hm. If I'm not mistaken, you already fixed this before. Maybe it came back as a result of a rebase. The test should help. Thank you!

To avoid the test duplication it can be placed in skikoTest source set (similar to skikoTest in ui:ui module, or foundation: foundation)

Reply:
It was 2 bugs before.
The first time I fixed `until endIndex` (instead of `.. endIndex`).
And, now, I found `-startIndex`

Reply:
Moved to skikoTest